### PR TITLE
Add timestamp tracking

### PR DESCRIPTION
**Pull request by esigler**
Show a relative time for when the lock was taken, to avoid dealing with timezones...er, I mean to be more humane.  :)

CC @dougbarth @theckman for feedback

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -3,10 +3,13 @@ FileName:
...
 AbcSize:
-  Max: 28
+  Max: 30
 
 MethodLength:
-  Max: 17
+  Max: 20
+
+ClassLength:
+  Max: 105
```
Comment (esigler) [inline .rubocop.yml]: I'll make the robot happy later.

File: lita-locker.gemspec [hunk 2]
```diff
@@ -16,6 +16,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'lita', '>= 4.0.1'
   spec.add_runtime_dependency 'redis-objects'
+  spec.add_runtime_dependency 'actionview'
```
Comment (dougbarth) [inline lita-locker.gemspec]: That's a pretty big dependency. :smile:

**Comment by dougbarth** [comment]
:+1:

File: lita-locker.gemspec (hunk 2 above)
Comment (esigler) [inline lita-locker.gemspec]: Aww c'mon, doesn't everyone want to install nokogiri?  :laughing:  Updated to use time-lord.

**Pull request merged**